core/templates/dev/head/pages/email_dashboard/EmailDashboard.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for oppia email dashboard page.
+ */
+
+oppia.controller('EmailDashboard', [
+  '$scope', '$http', function($scope, $http) {
+    var QUERY_DATA_URL = '/emaildashboarddatahandler';
+
+    // Store current cursor.
+    $scope.currentCursor = null;
```

Comment:
Yep, that sounds good to me.

One other thought: I suggest that you create an Angular service to store the data, and keep the controller minimal. This is good practice because a service can be more easily tested, and you won't end up mixing all the UI stuff with the "model" layer and logic. We're planning to move gradually towards this structure in all our frontend pages that have significant logic components, in order to make the code easier to maintain going forward.

Here's an [example](https://github.com/oppia/oppia/blob/simple-editor-2/core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js) that shows how I did this recently for a simpler version of the editor that we're working on. Note that `data` is a private variable that maintains the "state" of the page, and the service exposes a bunch of helper methods like getTitle(), saveTitle(), addNewQuestion() etc. You could do something similar here. In addition, it would also be good to have a Query with a setStatus() method, similar to how we have Question objects [here](https://github.com/oppia/oppia/blob/simple-editor-2/core/templates/dev/head/domain/exploration/QuestionObjectFactory.js) -- you can put that in domain/email_dashboard.

(I realize that this is a bit more work at the outset, but I think it would be a good idea to try and practice making the frontend clean and maintainable from the beginning.)